lib/server/ForwardClientRequest.js

```diff
@@ -1,31 +1,37 @@
 "use strict";
 
-var Promise = require("bluebird");
```

Comment:
GitHub seems to think this file is being moved. That doesn't seem right to me. This seems like one file was removed and another added.